decidim-proposals/spec/system/collaborative_drafts_spec.rb

```diff
@@ -101,10 +105,15 @@
         let!(:component) do
           create(:proposal_component,
                  :with_creation_enabled,
-                 :with_geocoding_and_collaborative_drafts_enabled,
```

Comment:
is this trait used somewhere else? if no, can we remove it from the factory?

Reply:
I have searched the source code and it seems that its not used anymore, so I'll remove it as you suggest.